test/users_list.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest'),
+  app = require('../app'),
+  dictum = require('dictum.js');
+
+describe('GET /users', () => {
+  test('Test get users list. It should respond with code 200', () => {
+    const agent = request(app);
+    const user = {
+      name: 'ArquiMedes Galileo',
+      last_name: 'Alighieri',
+      email: 'arquimedes.ali@wolox.ar',
+      password: '123e123_rw'
+    };
+    return agent
+      .post('/users')
+      .send(user)
+      .then(created => {
+        expect(created.statusCode).toBe(201);
+        return agent
```

Comment:
Take care with the Promise chain hell. Return and close identity, so you don't have to ident to the right every time you use agent